build.gradle.kts

```diff
@@ -59,4 +59,5 @@ subprojects {
 
 tasks.register("clean", Delete::class) {
     delete(rootProject.buildDir)
+    delete(file("$rootProject/compose-metrics"))
```

Comment:
Could add it to .gitignore instead? Might be useful to keep the results between cleans